service-loadbalancer/build/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM    alpine:3.2
+
```

Comment:
can you please reduce the spaces in this file? (club parts off it into logically equivalent sections)